lib/KCard/__test__/KCard.spec.js

```diff
@@ -0,0 +1,110 @@
+import { mount, createLocalVue } from '@vue/test-utils';
+import VueRouter from 'vue-router';
+import KCard from './../index.vue';
+
+const localVue = createLocalVue();
+localVue.use(VueRouter);
+
+const router = new VueRouter({
+  routes: [{ path: '/some-link' }],
+});
+
+function makeWrapper({ propsData = {}, slots = {} } = {}) {
+  return mount(KCard, { router, propsData, slots, localVue });
+}
+
+describe('index.vue', () => {
```

Comment:
The name of this Test Suite should be the same as the component we are testing, to more easily identify which component each test belongs to.

Reply:
I did not get you here

Reply:
Oh I was talking about renaming the describe block to 
```js
describe('KCard', () => {
```

To see in the  tests output that it is testing the KCard component. This is the output right now, and as you can see index.vue is not very descriptive (especially because this is logged among a lot of other tests).

```
  index.vue
    ✓ renders passed header level (20 ms)
    ✓ renders the correct accessibility  structure when title slot is passed (14 ms)
    ✓ renders the correct accessibility structure when title pass as props (6 ms)
    ✓ should not navigate on long click (508 ms)
    ✓ should navigate on quick click (109 ms)
    it renders slotted content
      ✓ renders slotted content with belowTitle slot (6 ms)
      ✓ renders slotted content with footer slot (6 ms)
```